pkg/framework/filelogger/filelogger.go

```diff
@@ -51,15 +52,17 @@ func New(config Config) (*FileLogger, error) {
 	return f, nil
 }
 
-func (f FileLogger) StartLoggingPod(namespace, name string) error {
+func (f FileLogger) StartLoggingPod(ctx context.Context, namespace, name string) error {
 	req := f.k8sClient.CoreV1().RESTClient().Get().Namespace(namespace).Name(name).Resource("pods").SubResource("log").Param("follow", strconv.FormatBool(true))
 
 	var readCloser io.ReadCloser
 	var err error
 
 	o := func() error {
 		readCloser, err = req.Stream()
-		if err != nil {
+		if apierrors.IsNotFound(err) {
+			return backoff.Permanent(microerror.Maskf(notFoundError, "pod %#q in namespace %#q", name, namespace))
```

Comment:
I think the `fmt` docs have still to be extended with the `%#q` placeholder suggestions.